Build with currently supported grpcio package

I can't build the current version of *grpc4bmi* with the currently supported *grpcio* conda package. Instead, I'm using the older *grpc-cpp* and *abseil-cpp* packages. *grpc-cpp* tops out at version 1.51.1, while *abseil-cpp* ends at version 20220623.0, so both are a few years out of date.

I think the *grpc4bmi* build has to be updated to switch to *grpcio*. 

Links:

* https://anaconda.org/conda-forge/grpcio
* https://anaconda.org/conda-forge/grpc-cpp

This issue https://github.com/abseil/abseil-cpp/issues/1624 may be blocking. I see it when trying to build with `grpcio`.
```console
/opt/conda/bin/../lib/gcc/x86_64-conda-linux-gnu/14.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: ../libgrpc4bmi.so: undefined reference to `absl::lts_20250512::Mutex::Dtor()'
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -25,7 +25,7 @@ RUN git submodule update --init
 WORKDIR ${prefix}/cpp/_build
 RUN cmake .. -DCMAKE_INSTALL_PREFIX=${CONDA_DIR} -DCMAKE_CXX_STANDARD=17 && \
     make && \
-    ctest && \
+    ctest -V && \
     make install && \
     make clean
 

--- a/Dockerfile
+++ b/Dockerfile
@@ -6,7 +6,6 @@ LABEL org.opencontainers.image.url="https://hub.docker.com/r/csdms/grpc4bmi"
 LABEL org.opencontainers.image.source="https://github.com/csdms/grpc4bmi-docker"
 LABEL org.opencontainers.image.vendor="CSDMS"
 
-# See https://github.com/csdms/grpc4bmi-docker/issues/1
 RUN conda install -y \
     libgrpc \
     libabseil \

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,5 +1,5 @@
 # Build grpc4bmi from gRPC conda packages on a condaforge/miniforge3 base.
-FROM csdms/bmi:0.2.1
+FROM csdms/bmi:0.3.0
 
 LABEL org.opencontainers.image.authors="Mark Piper <mark.piper@colorado.edu>"
 LABEL org.opencontainers.image.url="https://hub.docker.com/r/csdms/grpc4bmi"

--- a/Dockerfile
+++ b/Dockerfile
@@ -8,8 +8,17 @@ LABEL org.opencontainers.image.vendor="CSDMS"
 
 # See https://github.com/csdms/grpc4bmi-docker/issues/1
 RUN conda install -y \
-    grpc-cpp \
+    grpcio \
+    "protobuf>=4,<5" \
+    "googleapis-common-protos>=1.5.5" \
+    grpcio-reflection \
+    grpcio-status \
     && conda clean --all -y
+
+    # "googleapis-common-protos>=1.5.5" \
+    # "protobuf>=4,<5" \
+    # "libabseil<20240200" \
+    # grpcio-reflection \
 
 # See https://github.com/csdms/grpc4bmi-docker/issues/2
 ENV base_url=https://github.com/csdms
@@ -17,8 +26,8 @@ ENV project=grpc4bmi
 ENV version="0.6.0-csdms"
 ENV prefix=/opt/${project}
 RUN git clone --branch v${version} --depth 1 ${base_url}/${project} ${prefix}
-WORKDIR ${prefix}
-RUN git submodule update --init
+# WORKDIR ${prefix}
+# RUN git submodule update --init
 WORKDIR ${prefix}/cpp/_build
 RUN cmake .. -DCMAKE_INSTALL_PREFIX=${CONDA_DIR} -DCMAKE_CXX_STANDARD=17 && \
     make && \

--- a/Dockerfile
+++ b/Dockerfile
@@ -20,8 +20,6 @@ ENV project=grpc4bmi
 ENV version="0.6.0-csdms"
 ENV prefix=/opt/${project}
 RUN git clone --branch v${version} --depth 1 ${base_url}/${project} ${prefix}
-WORKDIR ${prefix}
-RUN git submodule update --init
 WORKDIR ${prefix}/cpp/_build
 RUN cmake .. -DCMAKE_INSTALL_PREFIX=${CONDA_DIR} -DCMAKE_CXX_STANDARD=17 && \
     make && \

--- a/Dockerfile
+++ b/Dockerfile
@@ -8,17 +8,11 @@ LABEL org.opencontainers.image.vendor="CSDMS"
 
 # See https://github.com/csdms/grpc4bmi-docker/issues/1
 RUN conda install -y \
-    grpcio \
+    libgrpc \
+    libabseil \
     "protobuf>=4,<5" \
-    "googleapis-common-protos>=1.5.5" \
-    grpcio-reflection \
-    grpcio-status \
+    libprotobuf \
     && conda clean --all -y
-
-    # "googleapis-common-protos>=1.5.5" \
-    # "protobuf>=4,<5" \
-    # "libabseil<20240200" \
-    # grpcio-reflection \
 
 # See https://github.com/csdms/grpc4bmi-docker/issues/2
 ENV base_url=https://github.com/csdms
@@ -26,8 +20,8 @@ ENV project=grpc4bmi
 ENV version="0.6.0-csdms"
 ENV prefix=/opt/${project}
 RUN git clone --branch v${version} --depth 1 ${base_url}/${project} ${prefix}
-# WORKDIR ${prefix}
-# RUN git submodule update --init
+WORKDIR ${prefix}
+RUN git submodule update --init
 WORKDIR ${prefix}/cpp/_build
 RUN cmake .. -DCMAKE_INSTALL_PREFIX=${CONDA_DIR} -DCMAKE_CXX_STANDARD=17 && \
     make && \

